scripts/gce_integration_test_metadata.in

```diff
@@ -0,0 +1,24 @@
+#cloud-config
+
+users:
+- name: testapp
+  uid: 2000
+
+write_files:
+- path: /etc/systemd/system/testapp.service
+  permissions: 0644
+  owner: root
+  content: |
+    [Unit]
+    Description=Start a simple docker container
+
+    [Service]
+    Environment="HOME=/home/testapp"
+    ExecStartPre=/usr/share/google/dockercfg_update.sh
+    ExecStart=/usr/bin/docker run --rm -u 2000 --name=testapp ${TESTED_IMAGE}:latest /bin/sleep 3600
```

Comment:
You need to expose the HTTP port here. Something like `-p 80:8080`. Also, you'll want to remove `/bin/sleep 3600` and rely on the default `CMD`.